cli/tools/task.rs

```diff
@@ -299,8 +299,10 @@ impl ShellCommand for NpmPackageBinCommand {
       },
     ];
     args.extend(context.args);
-    let executable_command =
-      deno_task_shell::ExecutableCommand::new("deno".to_string());
+    let executable_command = deno_task_shell::ExecutableCommand::new(
+      "deno".to_string(),
```

Comment:
Just making sure - specifying `deno` here will use the current executable, not look up `deno` on the `PATH`?

Reply:
Yeah, it's for the display name in error messages.

Reply:
By the way, this is also a bug fix because previously it wouldn't use the current executable.